### DISCUSSION TITLE
mydumper gets after hung after at somepoint

### DISCUSSION THREAD

**Discussion by asathvik** [mydumper/mydumper#1224] [General]
mydumper gets after hung after at somepoint.  these are the current configurations and command which i am using it

General Compute 16 Core Large Single SSD
Memory 128GB 
CPU 16
threads_dump=16

`mydumper --defaults-file $defaultsfile --outputdir=$outputdir --threads=$threads_dump --rows=100000 --regex '^(?!(mysql\.|test\.|performance_schema\.|information_schema\.|common_schema\.|sys\.|mysql_innodb_cluster_metadata\.))' -G -E -R -c -e --less-locking --skip-definer --trx-consistency-only -v 1 --logfile $logfilepath/mydumper_$myname.log`

Appreciate the response at the earliest

**Comment by davidducos** [comment]
Hi @asathvik can you test with latest release? https://github.com/mydumper/mydumper/releases/tag/v0.15.1-1

**Comment by asathvik** [comment]
hi @davidducos 
looks same issue even with latest version.
"i could see some strange" Is this causing the problem, how do we resolve this?
`64236 | system user        |                                                    | mysql                  | Query   |  2159 | Waiting for backup lock                     | NULL`

**Reply by davidducos** [reply]
Why don't you create a bug and fill all the information? As if you are using Percona Server I can tell you to use --no-backup-locks or even --no-locks. Can you share at least the log?

**Reply by asathvik** [reply]
even using locks it didn't help. my backup size are range 750 - 2 TB. 
Unfortunally i have enabled logging only for errors.

I see, mydumper v0.15.1-1 is. built against MySQL 5.7.42-46 with SSL support. Does this version support mysql community 8.0.33 version?

**Reply by asathvik** [reply]
release=$(curl -Ls -o /dev/null -w %{url_effective} https://github.com/mydumper/mydumper/releases/latest | cut -d'/' -f8)
yum -y install https://github.com/mydumper/mydumper/releases/download/${release}/mydumper-${release:1}.el8.x86_64.rpm

Its downloading.. 
mydumper-0.15.1-1.x86_64

mydumper --version
**mydumper v0.15.1-1, built against MySQL 5.7.42-46 with SSL support**

**Reply by davidducos** [reply]
I think that you are not going to have unsolvable issues using:
mydumper v0.15.1-1, built against MySQL 5.7.42-46 with SSL support
against MySQL 8.0.

**Reply by asathvik** [reply]
any fixes freezing/hung issues
?

**Reply by davidducos** [reply]
@asathvik how do you want me to help if you didn't provide valuable information? I already told you that you need to fill a bug providing all the information need to troubleshoot the issue. 
BTW, this is an open source project, if you need some level of support, I can offer some consulting time or you can use https://www.ringerhq.com/i/mydumper/mydumper.

**Comment by asathvik** [comment]
@davidducos  Please let me know your availablity for consulting time?  I am available now for 1hr

**Reply by davidducos** [reply]
@asathvik  Feel free to contact me by linkedin
